src/list-grid.scss

```diff
@@ -0,0 +1,261 @@
+@import './new-settings';
+@import './mixins';
+
+/*!
+.fd-grid-list
+*/
+$block: #{$fd-namespace}-grid-list;
+
+.#{$block} {
+  $item: #{$block}__item;
+  $more: #{$block}__more;
+
+  $fd-grid-list-border-radius: var(--sapElement_BorderCornerRadius);
+  $fd-grid-list-active-text-color: var(--sapList_Active_TextColor);
+  $fd-grid-list-item-box-shadow: var(--sapContent_Shadow0);
+  $fd-grid-list-mode-box-shadow: 0 0 0 0.125rem var(--sapList_SelectionBorderColor);
+  $fd-grid-list-item-selected-background-color: var(--sapList_SelectionBackgroundColor);
+  $fd-grid-list-item-active-background-color: var(--sapList_Active_Background);
+  $fd-grid-list-item-hover-background-color: var(--sapList_Hover_Background);
+  $fd-grid-list-highlight: (
+    'success': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'error': ('backgroundColor': var(--sapErrorBorderColor)),
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  &__item {
+    @include fd-flex(column);
+    @include fd-fiori-focus();
+
+    background-color: var(--sapList_Background);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    height: 100%;
+    position: relative;
+
+    &--body {
```

Comment:
This is not a modifier class. You are using it like this:
<img width="496" alt="Screen Shot 2020-11-12 at 12 07 32 PM" src="https://user-images.githubusercontent.com/39598672/98971850-b3eea300-24df-11eb-9b14-023b13578567.png">

It's a separate div element, it's not modifying the properties of the `fd-grid-list__item--body`. You can name it `fd-grid-list__item-body` and move it out of the `fd-grid-list__item` block. It will also need a reset.

Reply:
fixed